packages/pass-style/src/passStyleOf.js

```diff
@@ -81,28 +81,28 @@ const makePassStyleOf = passStyleHelpers => {
    * structures, so without this cache, these algorithms could be
    * O(N**2) or worse.
    *
-   * @type {WeakMap<Passable, PassStyle>}
+   * @type {WeakMap<WeakKey, PassStyle>}
```

Comment:
Do we have a TypeScript type corresponding with our Key concept (Passable structure in which each leaf is a Passable primitive value or a Remotable)?

Reply:
That reminds me: Key is another motivation for parameterizing Passable: `Passable<Remotable, never>`

PureData is `Passable<never, never>` as discussed earlier